src/spaceone/inventory/service/cloud_service_service.py

```diff
@@ -454,101 +454,85 @@ def _make_cloud_service_type_key(resource_data):
     def _make_region_key(resource_data, provider):
         return f'{resource_data["domain_id"]}.{provider}.{resource_data["region_code"]}'
 
-    def _change_metadata_path(self, metadata):
-        plugin_id = self.transaction.get_meta('plugin_id')
-
-        if plugin_id:
+    @staticmethod
+    def _convert_metadata(metadata, provider):
+        if provider:
             metadata = {
-                plugin_id: copy.deepcopy(metadata)
+                provider: copy.deepcopy(metadata)
             }
         else:
             metadata = {
-                'MANUAL': copy.deepcopy(metadata)
+                'custom': copy.deepcopy(metadata)
             }
 
         return metadata
 
-    def _get_collection_info(self, collection_info=None):
-        if collection_info is None:
-            collection_info = {}
+    def _get_collection_info(self, provider, collections: CollectionInfo = None):
+        if collections is None:
+            collections = []
 
         collector_id = self.transaction.get_meta('collector_id')
         secret_id = self.transaction.get_meta('secret.secret_id')
         service_account_id = self.transaction.get_meta('secret.service_account_id')
 
-        all_collectors = collection_info.get('collectors', [])
-        all_service_accounts = collection_info.get('service_accounts', [])
-        all_secrets = collection_info.get('secrets', [])
+        new_collection = {
+            'provider': provider,
+            'collector_id': collector_id,
+            'secret_id': secret_id,
+            'service_account_id': service_account_id,
+            'last_collected_at': datetime.utcnow()
+        }
 
-        if collector_id:
-            all_collectors.append(collector_id)
+        collections.append(new_collection)
+        return collections
 
-        if service_account_id:
-            all_service_accounts.append(service_account_id)
+    @staticmethod
+    def _merge_collection_info(old_collection_info, new_collection_info):
```

Comment:
Currently this function seems to be deprecated.

Reply:
have reflected thank you.